PFCalEE/src/DetectorConstruction.cc

```diff
@@ -890,9 +1034,118 @@ void DetectorConstruction::buildHGCALFHE(const unsigned aVersion){
     m_caloStruct.push_back( SamplingSection(lThick,lEle) );
     m_minEta.push_back(minEta[51]);m_maxEta.push_back(m_maxEta0);
   }
+
+  //Jan2021 version
+  else if (aVersion==9){
+			/*
+    G4double airThick = 1.5*mm;
+    G4double pcbThick = 1.6*mm;
+    if (version_==v_HGCAL_v8){
+      //back of ecal
+      lThick.push_back(pcbThick);lEle.push_back("PCB");
+      lThick.push_back(airThick);lEle.push_back("Air");
+      lThick.push_back(pcbThick);lEle.push_back("PCB");
+      lThick.push_back(0.1*mm);lEle.push_back("Cu");
+      lThick.push_back(1*mm);lEle.push_back("SSteel");
+    }
+    //CE-H layers 1
+    lThick.push_back(40*mm);lEle.push_back("SSteel");
```

Comment:
Just to check: at the end of EE, there is 45mm of SSteel, definitely no duplication here ?

Reply:
right indeed front CE-H  should start with the end of EE as done in version 8 - removed the corresponding lines in the CE-E part

Reply:
Just with the fresh morning eye, I am wondering if we should still have this here for the version with only EE, to have properly the back-scattered particles from the material behind the last layer. It's probably really a detail though. After that, I think we should merge and tag, and Bruno could send the gamma production ?
Ah, just FYI, still need also to debug the scintillator : something is wrong with the eta-phi to x,y conversion, I need to fix this.

Reply:
ok so let's do it like this - if it's EE only versions we add the back-plate for v8 and v9,
otherwise for full calorimeter we put the back-plate in front CEH